### ISSUE TITLE
Fix browser component on Linux (Ubuntu variants, others?)

### ISSUE THREAD

**Issue by kattrali** [danlucraft/redcar#64]
XULRunner was removed from the package repositories for Ubuntu some time ago. Find a way to build [from source](https://ftp.mozilla.org/pub/mozilla.org/xulrunner/releases/) during the installation process or migrate to WebKit.

**Comment by supermarin** [comment]
My vote goes for webkit

**Comment by kattrali** [comment]
Agreed. We only specify to use the default browser component on OS X and Linux, which is WebKit on OS X and XULRunner on Linux [when creating HTML Tabs](https://github.com/redcar/redcar/blob/master/plugins/application_swt/lib/application_swt/html_tab.rb#L20). The [SWT FAQ](http://www.eclipse.org/swt/faq.php#browserlinux) specifies the version of `libwebkitgtk` or XUL required to make the browser component work; maybe we can add `libwebkitgtk` as an installation dependency for the Linux installer.

I don't have a test machine/VM for this right now though.

**Comment by supermarin** [comment]
Maybe an issue to create a vagrant Ubuntu VM?

**Comment by kattrali** [comment]
Herp derp. That would make sense. :)
